src/pages/suggestions/[suggestion].tsx

```diff
@@ -19,6 +18,7 @@ import Instrument from "@/components/suggestion/instrument"
 
 interface SuggestionProps {
   suggestion: Song
+  isEditable: boolean
 }
 
 const SuggestionPage: FC<SuggestionProps> = (props: SuggestionProps) => {
```

Comment:
I believe we tend to name the  props interface `ComponentProps` so would be neater to rename the interface to `SuggestionPageProps`

Could write this as `({suggestion, isEditable} : SuggestionProps)` so you don't have to call props.attribute but can just call the attribute directly

 PS: _I know this is not your code so you don't necessarily have to change it_

Reply:
Done, added in new commit (same for edit-suggestion)